controls/1_1_master_node_api_server.rb

```diff
@@ -17,8 +17,10 @@
 
 title '1.1 Master Node: API Server'
 
+apiserver = attribute('apiserver', default: kubernetes.hyperkube ? 'apiserver' : 'kube-apiserver', description: 'The name of the apiserver process')
```

Comment:
I think we should return the command from `kubernetes.apiserver` and abstract the decision making there. Similar to what we do for ssh or nginx https://github.com/dev-sec/nginx-baseline/blob/master/libraries/nginx_lib.rb#L24-L37

Reply:
See comment above